src/plugins/swc-plugin.js

```diff
@@ -0,0 +1,141 @@
+import Visitor from '@swc/core/Visitor.js';
+import swc from '@swc/core';
+
+class JSXImportAppender extends Visitor.default {
+	visitModule(e) {
```

Comment:
I guess a better way to tackle this would be to do the latter half of this in a `visitImport` declaration.